about.html

```diff
@@ -0,0 +1,100 @@
+                </a>
+                <div id="links">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#">Join Program</a>
+                </div>
+            </nav>
+            <button type="button" id="mobile-btn" onclick="showMenu()">
+                <span class="material-icons">menu</span>
+            </button>
+            <div id="mobile-menu" class="menu">
+                <div class="menu-content">
+                    <a id="close-menu" onclick="closeMenu()">
+                        <span class="material-icons">close</span>
+                    </a>
+    
+                    <a href="./index.html" onclick="closeMenu()">
+                        Home
+                    </a>
+    
+                    <a href="./about.html" onclick="closeMenu()">
+                        About
+                    </a>
+                </div>
+            </div>
+        </header>
+        
+        <section class="introduction aboutintro">
+            <h4>console.log("Hello World!")</h4>
+            <h1>BEST CODING BOOTCAMP, CodeFire</h1>
+            <span>We are a coding bootcamp with the main objective of forming excellent software developers all around the world, we have been doing this from 2016 and we hope we can continue in this amazing job helping others reach their dreams</span>
+            <h3>Send us a message if you have questions! We will be pleased to attend you <br> code@fire.com</h3>
+        </section>
+        <section id="about">
+            <h2>And now, some information about us</h2>
+            <section id="logo">
+                <h3>CodeFire's logo</h3>
+                <h5>The official CodeFire's Logo</h5>
+                <img src="./src/icon.png" alt="">
+            </section>
+            <section class="no-flex">
+                <h3>See past CodeFire's awards</h3>
+                <h5>These awards are proof of what we do is working and having an impact in the real world</h5>
+                <div id="awards">
+                <div class="award-wrapper"> 
+                    <img src="./src/award1.png" alt="">
+                    <h3 class="image-centered">2021 <br> Best Coding Bootcamp <br> at Swiftchup</h3>
+                </div>
+
+                <div class="award-wrapper"> 
+                    <img src="./src/award2.png" alt="">
+                    <h3 class="image-centered">2021 <br> Best Coding Bootcamp <br> at CareerKarma</h3>
+                </div>
+            </div>
+            </section>
+            <section class="partners aboutpartners">
+                <h2>Partner</h2>
+                <ul id="partners-wrapper">
+                    <li>Google</li>
+                    <li>Facebook</li>
+                    <li>Microverse</li>
+                    <li>Uber</li>
+                    <li>Twitter</li>
+                </ul>
+            </section>
+        </section>
+        <footer class="icon">
```

Comment:
**Footer Desktop Version**
- Needs to have a logo aligned to the left and text aligned to the right.

**Footer Mobile Version**
- Provide a Footer

Reply:
Hi, I would like to know which footer should I add to the mobile version? The design has no footer for mobile in home page, only in about page

Reply:
In mobile version isn't displaying the footer.